SparkConnector/sparkconnector/configuration.py

```diff
@@ -269,6 +269,15 @@ def configure(self, opts, ports):
         conf.set('spark.kubernetes.namespace', os.environ.get('SPARK_USER'))
         conf.set('spark.master', self._retrieve_k8s_master(os.environ.get('KUBECONFIG')))
 
+        # The image used by the Spark executor should be set by the spawner, as we need
+        # different images for different platform types
+        # Get the value of 'SPARK_K8S_EXECUTOR_CONTAINER_IMAGE_URL' environment variable
```

Comment:
Can we make the name of the variable more generic? Potentially we could use this for the physical Hadoop clusters too, right? Something like `SPARK_EXECUTOR_IMAGE`.